tests/run/round.pyx

```diff
@@ -0,0 +1,10 @@
+# mode: run
+
+cdef float test_round_float(float value):
```

Comment:
Please use Python functions and doctests to make sure the functions actually do their work. Example:
```suggestion
import cython

def test_round_float(value: cython.float):
    """
    >>> test_round_float(3.14)
    3.0
    """
```

If you stick to Python syntax, you can then also rename the file to `.py` to let the test runner execute it twice, once compiled, once in Python. That makes sure that both give the same results.

Reply:
Thanks.